### PR TITLE
don't use deprecated stub function from koji

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-12400

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- n/a JSON/YAML configuration changes are updated in the relevant schema
- n/a Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

**Review by hjmodi — APPROVED** [review]

File: koji-containerbuild.spec [hunk 1]
```diff
@@ -63,7 +63,8 @@ Requires:   python3-six
 %description builder
 Builder plugin that extend Koji to communicate with OpenShift build system and
 build layered container images.
-%endif # with_python3
+%endif
+# with_python3
```
Comment (MartinBasti) [inline koji-containerbuild.spec:67]:
```suggestion
# end with_python3
%endif
```

and you know my opinion about unrelated changes in the same commit :)

Comment (rcerven) [reply koji-containerbuild.spec]: changed and added in separate commit

**Review by mkosiarc — APPROVED** [review]

**Comment by rcerven** [comment]
rn updated
